planning/autoware_surround_obstacle_checker/param/surround_obstacle_checker_node_parameters.yaml

```diff
@@ -0,0 +1,57 @@
+surround_obstacle_checker_node:
+  pointcloud:
+    enable_check:
+      type: bool
+      description: Enable to check surrounding obstacle
+  object_types:
+    type: string_array
+    default_value: [unknown, car, truck, bus, trailer, motorcycle, bicycle, pedestrian]
```

Comment:
@mitukou1109 Is it possible to remove default values...?

And, I think you can add validation like this.
```yaml
    validation:
      subset_of<>: [[unknown, car, truck, bus, trailer, motorcycle, bicycle, pedestrian]]
```

https://github.com/PickNikRobotics/generate_parameter_library?tab=readme-ov-file#built-in-validators

Reply:
You can add `unique<>` as well.

Reply:
`object_types` and `obstacle_types` are for generating parameter maps. Parameters with such names do not exist.

https://github.com/PickNikRobotics/generate_parameter_library?tab=readme-ov-file#mapped-parameters

Reply:
Ah. I see.